packages/node/src/indexer/fetch.module.ts

```diff
@@ -101,15 +101,7 @@ import { UnfinalizedBlocksService } from './unfinalizedBlocks.service';
     },
     FetchService,
     BenchmarkService,
-    {
```

Comment:
I don't think this is right. Where does `dictionaryService.init();` get called now?

Reply:
https://github.com/subquery/subql/blob/5b0ed9fc413e6b6e7b104b542a22eeedb68ba34e/packages/node-core/src/indexer/dictionary.service.ts#L167

It is here now